nginx-app/recipes/configure.rb

```diff
@@ -63,6 +63,7 @@
       :doc_root           => ::EasyBib::Config.get_appdata(node, application, 'doc_root_dir'),
       :app_dir            => ::EasyBib::Config.get_appdata(node, application, 'app_dir')
     )
+    notifies :run, 'execute[configtest]', :immediately
```

Comment:
@till I do remember that both of you mentioned `Easybib.providers.nginx` as the designated location for this directive, but the `notifies :reload 'service[nginx]'` is also here and I believe that these two lines should go together as it enhances the readability of the Chef run when debugging problems.

I am pointing it out so you can call a final veto on it.